Sources/WalletConnectRelay/Dispatching.swift

```diff
@@ -20,7 +20,8 @@ final class Dispatcher: NSObject, Dispatching {
     private let logger: ConsoleLogging
     
     private let defaultTimeout: Int = 5
-
+    private var fallback = false
```

Comment:
could we describe the purpose of the variable here, so it will be easier to get context later?